packages/connect-explorer/README.md

```diff
@@ -5,4 +5,4 @@ This package serves as:
 -   an interactive documentation for [@trezor/connect](../connect) package
 -   a referential implementation of @trezor/connect-web in "popup mode"
 
-`yarn dev` - runs dev server on port 8082
+`yarn dev` - runs dev server on port 8088
```

Comment:
maybe lets extend it with info that `connect-iframe` and `connect-popup` are also served all together at that port?

Reply:
Great idea to improve it! Please have a look at new commit [8863976](https://github.com/trezor/trezor-suite/pull/5571/commits/88639769b6885ef8fb380a5aa27a342ba603ff4a) and let me know if you have any other idea.

Reply:
👌 perfect